ntbs-service/DataMigration/NotificationImportService.cs

```diff
@@ -131,16 +131,17 @@ private async Task<List<ImportResult>> ImportNotificationGroupsAsync(PerformCont
 
         private async Task<ImportResult> ValidateAndImportNotificationGroupAsync(PerformContext context, string requestId, List<Notification> notifications)
         {
-            var patientName = notifications.First().PatientDetails.FullName;
-            var importResult = new ImportResult(patientName);
+            var patientFullName = notifications.First().PatientDetails.FullName;
+            var patientInitials = $"{notifications.First().PatientDetails.GivenName[0]}.{notifications.First().PatientDetails.FamilyName[0]}.";
+            var importResult = new ImportResult(patientFullName);
 
-            _logger.LogInformation(context, requestId, $"{notifications.Count} notifications found to import for {patientName}");
+            _logger.LogInformation(context, requestId, $"{notifications.Count} notifications found to import for patient with initials {patientInitials}");
```

Comment:
As discussed, I think it would be better if we used the Legacy ID of the first notification in the group here because it's both less ambiguous, and definitely not identifiable.